src/main/java/de/tum/in/www1/artemis/service/FileUploadSubmissionService.java

```diff
@@ -343,8 +179,8 @@ public FileUploadSubmission findOneWithEagerResultAndFeedback(Long submissionId)
      * @param submissionId the id of the submission that should be loaded from the database
      * @return the file upload submission with the given id
      */
-    public FileUploadSubmission findOneWithEagerResult(Long submissionId) {
-        return fileUploadSubmissionRepository.findByIdWithEagerResult(submissionId)
+    public FileUploadSubmission findOneWithEagerResultAndAssessor(Long submissionId) {
+        return fileUploadSubmissionRepository.findByIdWithEagerResultAndAssessor(submissionId)
                 .orElseThrow(() -> new EntityNotFoundException("File Upload submission with id \"" + submissionId + "\" does not exist"));
     }
```

Comment:
Again, this can be moved to a superclass

Reply:
Thank you for the suggestion. Moved it!